SemanticScribunto.php

```diff
@@ -50,7 +50,7 @@ public static function load() {
 	 */
 	public static function initExtension() {
 
-		define( 'SMW_SCRIBUNTO_VERSION', '1.0.0-alpha' );
+		define( 'SMW_SCRIBUNTO_VERSION', '1.0' );
 
```

Comment:
For versioning we follow the semver, which means the full 1.0.0 number. Further more doing a tidy and a release prep should be two distinct tasks.